src/services/document-api-client.js

```diff
@@ -61,6 +61,24 @@ class DocumentApiClient {
       .accept('json')
       .then(res => res.body);
   }
+
+  archiveDocument(documentKey) {
+    return this.httpClient
+      .post(`/api/v1/docs/${documentKey}/archive`)
```

Comment:
Not really restful, but I guess we don't care that much.

Reply:
Not all our endpoints are completely REST-ful, it's a matter of convenience with this one. Making this fully RESTful would mean quite some changes 😬